sources/packages/web/src/services/http/UserApi.ts

```diff
@@ -56,4 +57,35 @@ export class UserApi extends HttpBaseClient {
       throw error;
     }
   }
+
+  public async getinstitutionUser(
+    headers?: any,
+  ): Promise<InstitutionUserDetailsDto> {
+    try {
+      const response = await this.apiClient.get(
+        "users/institutionUser",
+        headers || this.addAuthHeader(),
+      );
+      return response.data as InstitutionUserDetailsDto;
+    } catch (error) {
+      this.handleRequestError(error);
+      throw error;
+    }
+  }
+
+  public async updateInstitutionUser(
+    data: InstitutionUserDetailsDto,
+    headers?: any,
+  ): Promise<void> {
+    try {
+      const response = await this.apiClient.patch(
```

Comment:
i think const response is not used inside the scope, you can remove it

Reply:
Yes. The const has been removed